website_product_document_download_counter/readme/DESCRIPTION.md

```diff
@@ -0,0 +1,7 @@
+This module allows you to track how many times product-related documents
+are downloaded from the Odoo website.
+
+Main features:
+
+- Automatic logging of downloads for documents attached to products.
```

Comment:
- Automatic logging the number of downloads of the documents attached to products.